inst/templates/package/validation_plan_package.Rmd

```diff
@@ -0,0 +1,77 @@
+
+
+## Testing Plan
+A Testing Plan document will be used to outline how tests will be developed, executed and recorded to ensure that the Requirements outlined in the Requirements Specification document are met. Testing in both development and Validation phases will be conducted using both manual and automatic tests as described in the Testing Plan document. Each test will have a unique identifier. Test results will be formally documented in a Testing Results document. Testing will occur throughout development and again during Validation at which point results will be formally documented and submitted for Validation with the date and time each test was run along with the attestation of the tester.
+
+
+## Traceability Matrix
+A Traceability Matrix will be used to match all tests stated in the Testing Results document to the Requirements outlined in the Requirement Specification document.
+
+
+## Validation Summary
+A Validation Summary document will be developed after Validation is completed to outline the results of the Validation process. All documents involved in the Validation process will be certified by the respective parties involved in their development. Validation certification will be provided by the following roles:
+
+* Tester: Person who conducts the testing set forth in the Testing Plan, documented in the Testing Results document and matched to the User Story Requirements summarized in the Requirements Specification through the Traceability Matrix. 
+* Author: Person who creates the Validation documentation as follows.
+    + Validation Plan: Product Manager
+    + Requirements Specification: Business Analyst
+    + Testing Plan: Business Analyst
+    + Testing Results: Tester
+    + Traceability Matrix: Business Analyst
+    + Validation Summary: Product Manager
+    + Release Notes: Product Manager
+    + Change Management Form: Product Manager
+* Reviewer: Person who reviews all Validation documentation for accuracy and completeness. This will generally be the Product Manager.
+* Approver: Quality Assurance representative responsible for approving the Validation documents and therefore certifying that the Release has been Validated. 
+
+
+## Release Notes
+Release Notes identifying all of the features represented by the User Stories developed in this release and all bugs addressed in this release will be provided along with the Release.
+
+```{r}
+format_release_changes(params$release_notes)
```

Comment:
I like this. Much cleaner, and also whatever changes we make to this will be self-contained in this function.